### PR TITLE
Update LADA

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
LADA

**Issue:** #10073

**Similarweb rank:** [20713](https://similarweb.com/website/lada.ru)

### Checklist
- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
Icon & colour from SVG in [source](https://www.lada.ru) header. Or should we stick with orange for the color?

### CODE REVIEW

**Comment by adamrusted** [comment]
@PeterShaggyNoble - I'm unable to open the source URL - could you share the SVG code here?

**Comment by PeterShaggyNoble** [comment]
Source path below, @adamrusted.
```
M38.4038 0C19.4992 0 3.78482 5.33987 0.615206 12.3699C0.222725 13.2053 0.0176886 14.112 0.0136719 15.0299C0.017977 15.9358 0.225887 16.8299 0.622917 17.65C0.622917 17.6749 0.648624 17.6998 0.658906 17.7248C0.669189 17.7497 0.71289 17.837 0.741167 17.8918C4.19098 24.7922 19.746 30 38.4167 30C59.622 30 76.8094 23.284 76.8094 15C76.8094 6.71597 59.6091 0 38.4038 0ZM35.5581 3.176C36.9025 4.40252 36.0722 7.06997 33.5016 10.7545C30.643 14.8355 27.8975 17.0766 24.6354 17.8993L31.7253 2.46801H31.7535C33.5658 2.38823 34.792 2.48546 35.5581 3.176ZM51.6427 18.2184C51.604 18.1936 51.5722 18.1599 51.5502 18.1203C51.5282 18.0808 51.5167 18.0365 51.5167 17.9915C51.5165 17.9286 51.5385 17.8674 51.5793 17.8184C51.62 17.7694 51.6769 17.7354 51.7404 17.7223L54.9357 17.0791C54.975 17.0709 55.0114 17.0527 55.0412 17.0264C55.071 17.0002 55.0931 16.9667 55.1053 16.9295L55.6195 15.4113C55.6296 15.3794 55.6322 15.3456 55.627 15.3125C55.6218 15.2794 55.6089 15.2479 55.5894 15.2204C55.5699 15.1928 55.5442 15.1699 55.5142 15.1533C55.4842 15.1367 55.4508 15.1269 55.4164 15.1246L49.4448 14.7956C49.2407 14.786 49.0389 14.8401 48.8689 14.9501L46.4757 16.5356C46.2679 16.6715 46.1057 16.8633 46.0088 17.0874C45.912 17.3115 45.8848 17.5582 45.9307 17.7971C46.599 21.2872 45.2854 23.1843 38.3987 24.0095C29.0106 25.1313 21.8282 24.1466 19.692 23.7876L19.6226 23.7751C19.471 23.7493 19.3148 23.7733 19.1789 23.8433C19.0429 23.9133 18.9349 24.0253 18.872 24.1615L18.1933 25.6423C11.124 23.5408 6.21666 20.3548 5.03416 16.7052C5.03416 16.6927 5.03416 16.6827 5.03416 16.6728C5.00074 16.5705 4.97246 16.4658 4.94418 16.3611V16.3387C4.83134 15.8983 4.77437 15.4463 4.77452 14.9925C4.77452 8.93718 14.9697 3.82666 28.9155 2.22121H28.9644C28.9644 2.22121 21.8231 17.7796 20.6072 20.4196C20.5882 20.4616 20.5801 20.5075 20.5837 20.5533C20.5874 20.599 20.6025 20.6432 20.628 20.6819C20.6534 20.7206 20.6882 20.7526 20.7294 20.7751C20.7706 20.7977 20.8169 20.81 20.8642 20.811H21.2601C25.4503 20.9531 30.5145 21.1027 36.8434 19.5446C43.5682 17.8868 48.6453 13.1278 50.5887 9.66511C52.5604 6.15007 51.4319 3.45022 51.2134 2.52534C63.6219 4.42995 71.5601 9.22885 71.7349 14.8105C71.6901 15.9496 71.3808 17.0642 70.83 18.0713C68.501 21.9553 61.8816 25.1762 53.072 26.9063C56.5398 23.102 53.8252 19.6418 51.6427 18.2259V18.2184Z
```

**Comment by adamrusted** [comment]
@PeterShaggyNoble - I'd argue the orange is still the brand colour, see the branding on their brick and mortar locations as well as the primary colour used for buttons etc on the site.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7742,9 +7742,14 @@
             "guidelines": "https://forums.ni.com/t5/NI-Partner-Network/New-Partner-Co-Marketing-Style-Guide/ba-p/3786987"
         },
         {
-            "title": "Lada",
-            "hex": "ED6B21",
-            "source": "https://www.lada.ru/priora/sedan/accessories.html"
+            "title": "LADA",
+            "hex": "4C5865",
```
Comment (adamrusted) [inline _data/simple-icons.json:7746]:
```suggestion
            "hex": "ED6B21",
```

**Review by adamrusted — APPROVED** [review]
Just revert the hex and this is good to go!

**Pull request merged**